lib/src/schedule_parse_exception.dart

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) 2022, Agilord. All rights reserved. Use of this source code
+// is governed by a BSD-style license that can be found in the LICENSE file.
+
+/// Exception thrown when a cron data does not have an expected
+/// format and cannot be parsed or processed.
+class ScheduleParseException extends FormatException {
```

Comment:
There is very little reason for this to live in a separate file, it could just be in the constraint_parser.dart

Reply:
I'm not familiar with the design philosophy of this package, so I will leave it to you to change the contents of the commit! Thanks for the merge anyway :)